compiler/test/stdlib/number.test.gr

```diff
@@ -309,3 +369,48 @@ assert Number.cos(-1) > 0.5403 && Number.cos(-1) < 0.5404
 assert Number.cos(15) < -0.7596 && Number.cos(15) > -0.7597
 assert Number.isNaN(Number.cos(Number.infinity))
 assert Number.isNaN(Number.cos(Number.nan))
+
+// Number.tan
+assert Number.tan(1) > 1.5574 && Number.tan(1) < 1.5575
+assert Number.tan(-1) > -1.5575 && Number.tan(-1) < -1.5574
+assert Number.tan(15) < -0.8559 && Number.tan(15) > -0.8560
+assert Number.isNaN(Number.tan(Number.infinity))
+assert Number.isNaN(Number.tan(Number.nan))
+
+// Number.Gamma
+assert Number.gamma(10.5) == 1133278.3889487908
+assert Number.gamma(10) == 362880
+assert Number.gamma(1) == 1
+assert Number.gamma(0.5) == 1.7724538509055164
+assert Number.gamma(0) == Number.infinity
+assert Number.gamma(-0.5) == -3.544908144938388
+assert Number.gamma(-1) == Number.infinity
+assert Number.gamma(-10) == Number.infinity
+assert Number.gamma(-10.5) == -2.6401221505732836e-7
```

Comment:
I don't have a quick way to check these

Reply:
You can check against Pythons `math.gamma` do note that `gamma` is an approximation it also depends on `sin` which is an approximation so it differs slightly from python. I think in the future we could get closer to pythons awnser with a slightly different implementation and by using stirlings apprimiation when the number is less then 0.5 but this should be done in a seperate pr.